test_gh_config.yaml

```diff
@@ -2,8 +2,8 @@
 # 1 profile loaded with default speeds is defined for each vehicle type
 
 graphhopper:
-  datareader.file: test-data/kansas-city-extract-mini.osm.pbf
```

Comment:
I'm going to undo this change when we're ready to merge, as it breaks CI currently; the next change (queued up right after this is merged) will integrate this script properly into CI + will include this update to `test_gh_config`

Reply:
Did this, should be gtg with CI now